app/components/ui/Cards/CategoryCard.module.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../../styles/utils/_helpers.scss";
+
+.categoryCard {
+  display: flex;
+  flex-direction: row;
+  gap: $general-spacing-s;
+  background-color: $white;
+  padding: $general-spacing-lg;
+  height: auto;
+  width: 100%;
+  border-radius: $rounded-md;
+  box-shadow: 0px 1px 3px 0px rgba(112, 79, 29, 0.1),
+    0px 1px 2px -1px rgba(112, 79, 29, 0.1);
+  position: relative;
+  cursor: pointer;
+
+  @media screen and (min-width: $break-tablet-p) {
+    flex-direction: column;
+    height: 150px;
+  }
+
+  &:hover {
+    color: $color-brand-dark;
+  }
+
+  .categoryTitle a {
+    font-size: 1.25rem;
+    font-weight: 600;
+    color: $text-primary;
+    transition: none;
+
+    @media screen and (max-width: $break-tablet-p) {
+      font-size: 1.125rem;
+    }
+
+    &:hover {
+      color: $color-brand-dark;
+      text-decoration: none;
+    }
+
+    &::after {
```

Comment:
What is the purpose of this ::after section? If I'm not mistaken, it seems like it could be a straight forward flex div to me.

Reply:
thanks for the catch. I originally had the label as an h4 for styling and wanted to avoid wrapping it with an a tag